scripts/e2e_get_tools.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o errexit
```

Comment:
Since this script is intended to be used with `source ...` in case of error the user shell gets closed without seeing the error. Can it be removed?

Reply:
Done.
Chose to remove.